srcpkgs/zoom/template

```diff
@@ -16,6 +17,32 @@ noshlibprovides=yes
 noverifyrdeps=yes
 restricted=yes
 nopie=yes
+build_options="systemqt"
```

Comment:
You probably want `build_options_default="systemqt"` as well?

Reply:
If enabled by default, increase revision.

Reply:
Since this might break things, I'd not enable it by default.